csatar-theme/pages/design-system.htm

```diff
@@ -0,0 +1,33 @@
+title = "CSATÁR design system"
+url = "/design_system"
+==
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    {% partial 'site/meta' %}
+</head>
+
+<body>
+    {% partial 'design-system/title' %}
+    {% partial 'design-system/colors' %}
+    {% partial 'design-system/horizontal-separator' %}
+    {% partial 'design-system/typography' %}
+    {% partial 'design-system/horizontal-separator' %}
+    {% partial 'design-system/buttons' %}
+    {% partial 'design-system/horizontal-separator' %}
+    {% partial 'design-system/icons' %}
+    {% partial 'design-system/horizontal-separator' %}
+    {% partial 'design-system/cards' %}
+    {% partial 'design-system/horizontal-separator' %}
+    {% partial 'design-system/forms' %}
+
+
+
+    {# Copyright #}
+    <div class="bg-gray align-items-start p-5 mt-2">
+    <p class="p-title" style="color:#939393;">© Copyright CSATÁR 2022</p>
```

Comment:
Itt van valami oka, hogy a style inline van?

```suggestion
    <p class="p-title" style="color:#939393;">© Copyright CSATÁR {{ 'now' | date('Y') }}</p>
```
Ahol az aktuális évet kell megjeleníteni, lehet használni a twig beépített funkcióját: `{{ 'now' | date('Y') }}`